package.json

```diff
@@ -66,7 +66,7 @@
     "tsx": "^3.12.7",
     "type-fest": "^3.13.1",
     "typedoc": "^0.23.28",
-    "typescript": "^5.1.6"
+    "typescript": "^5.2.2"
```

Comment:
I intentionally did not use this version, please do not change it.

Reply:
i didnt change it. i just did pnpm up. sorry